### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by Ritesh1137** [maximevaillancourt/digital-garden-jekyll-template#184] [Q&A]
This Jekyll theme is very interesting for creating a digital garden, but can you add some information about how to customize the layout and structure of the theme in the ReadMe documentation. I am looking to add a Nav bar, possibly a dark mode, a slighly altered notes layout and a projects section. I want a Jekyll themed digital garden to have [this UI](https://hugo-digital-garden-demo.netlify.app/)

**Comment by maximevaillancourt** [comment]
Thanks for reaching out. I don't provide support for template customization.

The template is based on HTML, CSS, and JavaScript. Learn how to use these tools ([MDN is an excellent resource](https://developer.mozilla.org/en-US/docs/Learn/Front-end_web_developer)), and start customizing your garden to your taste. You can also find inspiration in [the hundreds of public forks of this template](https://github.com/maximevaillancourt/digital-garden-jekyll-template/forks). Some people may have implemented what you're looking for.

Consider starting your exploration in the main CSS file here: https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/main/_sass/_style.scss
